### PR TITLE
Change the test contant in virtual_nic_stress file

### PR DESCRIPTION
**Pull request by yanglei-rh**
ID:1792862

Signed-off-by: Lei Yang <leiayang@redhat.com>

### CODE REVIEW

**Comment by yanglei-rh** [comment]
@PaulYuuu @kimi1978wy Could you please help me review it? Thanks a lot.

File: qemu/tests/virtual_nic_stress.py [hunk 1]
```diff
@@ -13,32 +12,30 @@ def run(test, params, env):
...
-                session = vm.wait_for_login(timeout=timeout)
-                pass
-        return session
+            output = process.system_output(ping_cmd, ignore_status=True,
+                                           timeout=flood_minutes * 60)
+        except process.CmdError:
+            err = "Do flood ping from host to guest failed"
+            err += "Command: %s," % ping_cmd
+            err += "output: %s." % output
+            test.fial(err)
```
Comment (kimi1978wy) [inline qemu/tests/virtual_nic_stress.py:38]: typo

File: qemu/tests/virtual_nic_stress.py [hunk 2]
```diff
@@ -66,8 +63,9 @@ def unload_stress(session):
 
     timeout = float(params.get("login_timeout", 240))
     vm = env.get_vm(params["main_vm"])
-    host_ip = utils_net.get_host_ip_address(params)
+    vm.verify_alive()
```
Comment (kimi1978wy) [inline qemu/tests/virtual_nic_stress.py:66]: I think `vm.wait_for_login(timeout=timeout)` is enough, no need to verify alive.

File: qemu/tests/virtual_nic_stress.py [hunk 3]
```diff
@@ -92,11 +91,12 @@ def unload_stress(session):
...
-                  " output: %s" % (status, output))
-
+    count_cmd = "ping %s -c %s" % (guest_ip, count)
+    try:
+        output = process.system_output(count_cmd, timeout=float(count) * 2)
+    except process.CmdError:
+        err = "Ping from host to guest failed"
+        err += "Command: %s," % count_cmd
+        err += "output: %s." % output
+        test.fial(err)
```
Comment (kimi1978wy) [inline qemu/tests/virtual_nic_stress.py:101]: typo

Comment (yanglei-rh) [reply qemu/tests/virtual_nic_stress.py]: @kimi1978wy Thanks for your review, I have changed them. Please help me review it again.

File: qemu/tests/virtual_nic_stress.py [hunk 4]
```diff
@@ -13,32 +12,30 @@ def run(test, params, env):
...
+        Do flood ping from host to guest
 
-        :param session: session to send flood ping
-        :param host_ip: the IP of the host
         """
         flood_minutes = int(params["flood_minutes"])
         logging.info("Flood ping for %s minutes", flood_minutes)
+        ping_cmd = "ping %s -f -q" % guest_ip
         try:
-            utils_net.ping(host_ip, flood=True,
```
Comment (PaulYuuu) [inline qemu/tests/virtual_nic_stress.py:34]: I think we can keep using `utils_net.ping`, with guest_ip and session=None

File: qemu/tests/virtual_nic_stress.py [hunk 5]
```diff
@@ -92,11 +90,12 @@ def unload_stress(session):
                           " Check if the network is still alive",
                           logging.info)
     count = params["count"]
-    timeout = float(count) * 2
-    status, output = utils_net.ping(host_ip, count, session=session,
```
Comment (PaulYuuu) [inline qemu/tests/virtual_nic_stress.py:96]: Same here.

Comment (yanglei-rh) [reply qemu/tests/virtual_nic_stress.py:96]: Hi @PaulYuuu Please help me review it again.Thanks.

**Review by PaulYuuu — APPROVED** [review]

**Comment by yanglei-rh** [comment]
Test Pass:
 (1/1) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.virtual_nic.with_stress.q35: PASS (2069.63 s)
(1/1) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.i386.io-github-autotest-qemu.virtual_nic.with_stress.q35: PASS (1934.83 s)

**Review by kimi1978wy — APPROVED** [review]
LGTM
Acked-by: Yu Wang <wyu@redhat.com>

 (1/2) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.i386.io-github-autotest-qemu.virtual_nic.with_stress.q35: PASS (1842.55 s)
 (2/2) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.virtual_nic.with_stress.q35: PASS (2319.27 s)

**Pull request merged**